sqlmesh/dbt/basemodel.py

```diff
@@ -289,10 +294,21 @@ def sqlmesh_model_kwargs(self, context: DbtContext) -> t.Dict[str, t.Any]:
         jinja_macros = model_context.jinja_macros.trim(
             self.dependencies.macros, package=self.package_name
         )
+
+        model_node: AttributeDict[str, t.Any] = AttributeDict(
```

Comment:
btw, I don't think you need to convert to `AttributeDict` since the validator within `JinjaMacros` should take of that.